### DISCUSSION TITLE
FileNotFoundError: No such file or directory: 'D:\\memilio\\data\\mobility/commuter_mobility.txt'

### DISCUSSION THREAD

**Discussion by Sandhya-105** [SciCompMod/memilio#1142] [Q&A]
Hi!

My name is Sandhya. I am a PhD student from TU Wien. I would like to use this data for my research but I encountered an error while accessing the dataset. Would it be possible for you to provide me with the dataset file? 

This happened when I tried to do the steps from memilio\cpp\simulations\data_generation.sh file

Thanks,
Sandhya

**Comment by mknaranja** [comment]
Dear @Sandhya-105, sorry for the late reply. We haven't seen your discussion item. We will come back to you soon, I need to get in touch with one of our data experts :)

**Reply by HenrZu** [reply]
Hi, we have looked at the problem and are actively working on a fix, which will soon be merged into the main.
